core/crawler-conf.yaml

```diff
@@ -43,8 +43,6 @@ parsefilters.config.file: "parsefilters.json"
 urlfilters.config.file: "urlfilters.json"
 
 parser.htmlmapper.classname: "org.apache.tika.parser.html.IdentityHtmlMapper"
-parser.ignore.outlinks.outside.host: true
```

Comment:
These params are easier to change in the general config than buried in the urlfilters.json, since your patch allows the filters to be configured from the global config why not leave them there?